client/my-sites/sidebar/static-data/domain-only-fallback-menu.ts

```diff
@@ -0,0 +1,36 @@
+import { translate } from 'i18n-calypso';
+
+const domainOnlyFallbackMenu = ( { siteDomain }: { siteDomain: string } ) => {
+	return [
+		{
+			icon: 'dashicons-admin-settings',
+			slug: 'manage-domain',
+			title: translate( 'Manage domain' ),
```

Comment:
I recently got [some feedback](https://github.com/Automattic/wp-calypso/pull/66501#discussion_r945901619) about using the "non-reactive" version version of `translate`. I'm wondering if this is another instance where it might be beneficial to use a reactive version.

Maybe @dlind1 can help clarify?

Reply:
I'd say that using reactive versions of translation functions should be the preferred way since otherwise changes to the locale (or delayed loading of translations) won't trigger a re-render and the user would be presented with a mix of translated and untranslated text. You can check the documentation [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/README.md).

Reply:
@fredrikekelund @dlind1  while typically the reactive version is often the preferred way, I think that in this case I would stick with the "non-reactive" version for the following reasons:

- this is consistent with all other units in similar contexts
- the fallback menu items are only surfaced briefly when the `admin-menu` endpoint hasn't returned, and replaced from the backend
- a cache is used when it exists